common/djangoapps/entitlements/models.py

```diff
@@ -11,7 +11,7 @@ class CourseEntitlement(TimeStampedModel):
     """
 
     user = models.ForeignKey(settings.AUTH_USER_MODEL)
-    uuid = models.UUIDField(default=uuid_tools.uuid4, editable=False)
+    uuid = models.UUIDField(default=uuid_tools.uuid4, editable=False, unique=True)
```

Comment:
do you need a migration for this?

Reply:
It should be in this PR.

Reply:
The file, common/djangoapps/entitlements/migrations/0003_auto_20171120_1432.py is included in the PR with the migration.